shared/actions/chat2/index.js

```diff
@@ -2261,7 +2258,7 @@ function* createConversation2(state, action) {
 }
 
 const createConversation = (state, action, afterActionCreator) => {
-  if (action.type === Chat2Gen.createConversation && flags.newTeamBuildingForChat) {
+  if (action.type === Chat2Gen.createConversation) {
```

Comment:
i think this can be cleaned up further. 1 it doesn't need to be a side effect of createconversation anymore. 2 we might not need it as a side effect of message privately if we can just delegate to createconversation2

Reply:
Okay, removed `createConversation` type from the action and simplified. It then looks similar to createConversation2, but not sure it's worth combining.